src/components/Property/RepairsHistory/RepairsHistoryView.js

```diff
@@ -7,19 +7,27 @@ import { getRepairsForProperty } from '../../../utils/frontend-api-client/repair
 import { sortObjectsByDateKey } from '../../../utils/date'
 
 const RepairsHistoryView = ({ propertyReference, tabName }) => {
+  const [pageNumber, setPageNumber] = useState(1)
   const [workOrders, setWorkOrders] = useState([])
+  const [arrayForHoldingworkOrders, setArrayForHoldingworkOrders] = useState([])
```

Comment:
Think this should be `arrayForHoldingWorkOrders`

Reply:
Can't we just call it `AllWorkOrders` ?

Reply:
I would go with
```
const [workOrdersPerPage, setWorkOrdersPerPage] = useState([]) // which will hold work orders per page
const [workOrders, setWorkOrders] = useState([]) // which will hold all work orders
```